client/.eslintrc

```diff
@@ -9,6 +9,7 @@
   "rules": {
     "no-debugger" : 1,
     "comma-dangle": 0,
+    "no-underscore-dangle": 0,
```

Comment:
This rule is poorly implemented. It triggers on any property, even ones that are defined by external modules which we don't control. If there was a case where a third party module used a leading underscore, it would not pass linting.

However, I got the same feedback from two different people, so I will switch it back and do a local override.